app/src/main/java/com/google/android/soundchecker/AudioEncoderDecoderActivity.kt

```diff
@@ -624,6 +630,7 @@ class AudioEncoderDecoderActivity : ComponentActivity() {
     }
 
     private fun runTest() {
+        mOutputTrackCount++ // Stop the previous audioStream
```

Comment:
Is there a race here?
Do we need to join() the player thread before we open a new output stream and codec?

Reply:
Added some comments.